Xwt/Xwt/Key.cs

```diff
@@ -27,6 +27,7 @@
 
 namespace Xwt
 {
+	[Flags]
```

Comment:
Good point, I never looked into `Key` values and I always assumed it was a flag in Gtk like modifier keys... sorry about that.